exporter/cat1_view_helper.go

```diff
@@ -241,3 +241,14 @@ func isNil(i interface{}) bool {
 func derefBool(i *bool) bool {
 	return *i
 }
+
+// RecordTarget is used for rendering the record_target template
+type RecordTarget struct {
```

Comment:
I chose to do it this way because CMS compatibility isn't a property of a patient. It only exists as a concept associated with a document. In other words, you could render a Cat 1 document for a patient with or without CMS compatibility. So it felt like the Record itself shouldn't know about it. @mattrbianchi